### ISSUE TITLE
Recaf crash

### ISSUE THREAD

**Issue by wagyourtail** [Col-E/Recaf#884]
Diagnostics:
```JavaFX on Recaf's classpath was too old```

![image](https://github.com/user-attachments/assets/0f8573de-f6f1-43c4-b495-d3bd96fa2a6e)

**Comment by xxDark** [comment]
Can you send your launcher log?
Also, try building launcher from [this fork](https://github.com/xxDark/Recaf-Launcher) and see if it helps.
See https://github.com/Col-E/Recaf-Launcher/pull/33 that is not merged yet.

**Comment by wagyourtail** [comment]
```
Looking in 'C:\Users\ruwag\AppData\Roaming\Recaf' for Recaf/dependencies...
Attempting to resolve JFX version in current classpath...
No JavaFX version class found in the current classpath
The current runtime bundles JavaFX that is too old (null).
- Try running with a JDK that has an up-to-date JavaFX bundled
- Or use a JDK that does not bundle JavaFX and let the launcher grab it for you
Failed launching Recaf: JavaFX on Recaf's classpath was too old
```

**Comment by Col-E** [comment]
The fork has been merged and released as launcher 0.8.0
